config/manifests/bases/datadog-operator.clusterserviceversion.yaml

```diff
@@ -21,6 +21,11 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: DatadogAgent Deployment with Datadog Operator.
+      displayName: Datadog Agent
+      kind: DatadogAgent
+      name: datadogagents.datadoghq.com
+      version: v1alpha1
```

Comment:
any idea where this version is coming from?

Reply:
it's a re-ordering, not an addition but the github UI makes it confusing. The real addition is the dashboard part that was somehow missed in last release I believe 
<img width="2203" alt="image" src="https://github.com/user-attachments/assets/13b34b07-8411-4b87-863d-ea919c4bd777">

Reply:
i saw that, but i'm wondering about DatadogAgent v1alpha1 version since that's been deprecated

Reply:
good catch, taking a look